knative-operator/Gopkg.toml

```diff
@@ -47,6 +47,10 @@ required = [
   name = "knative.dev/serving-operator"
   branch = "master"
 
+[[constraint]]
+  name = "knative.dev/eventing-operator"
+  branch = "master"
```

Comment:
I've been meaning to change this for the serving-operator, too. I think we need real versions here instead of a branch. And I think the version needs to match the CRD beneath olm-metadata/

Reply:
Can we change it both in a separate PR?

Reply:
Absolutely!